code/modules/clothing/chameleon.dm

```diff
@@ -168,6 +168,11 @@
 
 	target.icon = initial(picked_item.icon)
 
+	if(istype(target, /obj/item/stamp))
+		var/obj/item/stamp/S = target
+		var/obj/item/stamp/P = picked_item
+		S.stamp_color = initial(P.stamp_color)
```

Comment:
do you need to use `initial` here? I think it's better to use `P.stamp_color` instead. `initial` would probably break when chameleoning another chameleon stamp or something...

Reply:
I only got it to work by using `initial` was spitting strange error at me otherwise...

Reply:
all the other definitions in chameleon code use `initial` I dont believe this is incorrect code. I havent gotten any problems with extensive testing, includiung making the cham stamp chameleon.

Reply:
Oh, that's because `picked_item` is not an actual item, but rather just a path, I see..!